go.mod

```diff
@@ -4,17 +4,17 @@ go 1.20
 
 require (
 	filippo.io/edwards25519 v1.0.0
-	github.com/ChainSafe/chaindb v0.1.5-0.20221010190531-f900218c88f8
-	github.com/MarinX/monerorpc v1.0.5
+	github.com/ChainSafe/chaindb v0.1.5
+	github.com/MarinX/monerorpc v1.0.6
```

Comment:
That was a PR I did against MarinX's repo. (For incoming transfers, a feature we don't use.)

There are libp2p updates (only a single minor version increase, but backwards incompatible), that will involve code updates, so I didn't include them in this PR.